lib/embulk/command/embulk_new_plugin.rb

```diff
@@ -0,0 +1,87 @@
+module Embulk
+  def self.new_plugin(name, language, category)
+    require 'embulk/data/package_data'
+    require 'embulk/version'
+    require 'fileutils'
+
+    embulk_category = category
+    embulk_category = :input if category == :file_input
+    embulk_category = :output if category == :file_output
```

Comment:
We might want to add file_input and file_output plugins for Ruby in the future? NotImplementedError?